src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.received.txt

```diff
@@ -0,0 +1,2895 @@
+namespace GraphQL.Builders
```

Comment:
This file should not be pushed to git.

Reply:
Ugh. It needs to be ignored - remembered to not push it 100 times except this one. Can I add it to .gitignore?

Reply:
Ignored it and updated rule, though not sure why previous one wasn't ignoring it for me. Seemed right.

Reply:
When the api test is successful, this file is deleted automatically.